layouts/partials/head.html

```diff
@@ -20,6 +20,8 @@
     <link rel="shortcut icon" href="{{"/images/favicon.ico" | absURL}}" type="image/x-icon" />
     <link rel="icon" href="{{"/images/favicon.png" | absURL}}" type="image/x-icon" />
 
+    <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+
```

Comment:
Just wondering, is this necessary for every page? If so, can it be put in the footer, or be loaded asynchronously? Just so that the page load doesn't hang if this resource is slow...

Reply:
I've added a custom footer and moved the script there - could I ask you for verification? 

I've also modified the solution for sorting with a category. Now instead of displaying the `category` field in summary, we have grouping by a `category` like below. We can have multiple categories, so I suspect this is the reason it is displayed with `List["category"]`

![image](https://github.com/hs3city/hs3.pl/assets/86041155/dac84f12-4d62-4174-beaa-828045aaf67e)

Reply:
I guess my main question is: do we need this on every page? I can imagine we only need it on `/admin` -- but I don't know the system. As far as I can see, it does not place any tracking cookies, so at least we don't need a cookie banner if we put it on every page, but still it would be better not to put it on every page if we don't need it on every page :)

Reply:
And yes, category is a List object

Reply:
Tutorials I followed recommended including the script on all pages. However, this is used to authenticate to Decap CMS, so I also think that we should need this script only on `/admin`. 

I removed the custom footer and the authentication still works without issues.